src/common/components/repository-row/index.js

```diff
@@ -377,10 +376,8 @@ RepositoryRow.propTypes = {
     success: PropTypes.bool
   }),
   dispatch: PropTypes.func.isRequired,
-  registerNameIsOpen: PropTypes.bool
+  registerNameIsOpen: PropTypes.bool,
+  configureIsOpen: PropTypes.bool
 };
 
-// FIXME:
-// `connect` shouldn't be used just to add dispatch to props
-// this makes it much harder to test it as it get wrapped
```

Comment:
Sure, but in case of passing `dispatch` only I would probably prefer passing some callback from parent component or using `mapDispatchToProps`.